File: projects/project10/Java/FlyingBird.java
/* FlyingBird.java provides a class for the FlyingBird types
 *
 * Created by: Ethan Clark Project10
 * Date: April 25, 2016
 */

public class FlyingBird extends Bird
{

	/*******************************************************
	* Constructor for FlyingBird class                     *
	* Calls the name constuctor for the Bird parent class  *
	*******************************************************/
	public FlyingBird(String name)
	{
		super(name);
	}

	/*****************************************************************
	* movement() returns the movement type for the FlyingBird class  *
	* Receive: NONE                                                  *
	* Return: "flew"                                                 *
	*****************************************************************/
	public String movement()
	{
		return "flew";
	}

}

File: projects/project10/Ruby/FlyingBird.rb
# FlyingBird.rb | Defines the FlyingBird class which inherits attributes and methods
#   from the Bird superclass.
#
# Created by: Ethan Clark Project10
# Date: April 26, 2016
####################################################

require './Bird.rb'

class FlyingBird < Bird

	###########################################################
	# movement returns the move that a FlyingBird type does   #
	# Recieve: NONE                                           #
	# Return: flew, a String                                  #
	###########################################################
	def movement
		"flew"
	end

end
